mirrord/operator/src/crd/policy.rs

```diff
@@ -63,6 +63,10 @@ pub struct MirrordPolicySpec {
     /// the user config.
     #[serde(default)]
     pub fs: FsPolicy,
+
+    /// Fine grained control over network features like specifiying required http filters.
+    #[serde(default)]
+    pub network: NetworkPolicy,
```

Comment:
The network policy is now kind of distributed between `BlockedFeature`s and this.
I guess `env` and `fs` should have also been `BlockedFeature`s, but it's too late for that. But with network I find it a bit more confusing because there is already a `StealWithoutFilter` blocked feature.
I would expect the header requirement to be in a new, similar variant, that also includes a header filter specification.

Reply:
I feel like it's going to be more convoluted once a path filter is introduced and this is is already quite convoluted, here I like that it's simmilar to config